### PR TITLE
Added feature to run ATH against Selenium Grid instance

### PR DESCRIPTION
**Pull request by adini121**
Configured Jenkins ATH to run tests against a Selenium grid instance (hub). Tested locally as well as against Existing and tomcat controllers. Requirement: If the grid is running on different machine/host than the one running Jenkins instance, Jenkins URL needs to be made accessible to the grid. Much easier to do with "Existing" controller by setting JENKINS_URL (But requires Form-Element-Path plugin installed). 

Please let me know if this feature is helpful or needs modifications.

### CODE REVIEW

File: src/main/resources/seleniumGrid.properties [hunk 1]
```diff
@@ -0,0 +1,4 @@
+gridHubURL=http://yourGridURL:gridPort/wd/hub
+#example: gridHubURL=http://192.168.10.10:4444/wd/hub
+browserval=firefox
```
Comment (olivergondza) [inline src/main/resources/seleniumGrid.properties]: Is there any reason not to accept these values directly from properties passed to ATH proccess?

File: docs/SELENIUM-GRID.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+#Running ATH tests against Selenium Grid
+
+This feature enables us to test ATH against Selenium Grid
+
+##Requirements
+* JENKINS_URL must be accessible for the Selenium hub
+* If running against existing Jenkins instance using Jenkins Under Test (JUT) =EXISTING, A running Jenkins instance with pre-installed [Form-Element-Path plugin](https://wiki.jenkins-ci.org/display/JENKINS/Form+Element+Path+Plugin)
```
Comment (olivergondza) [inline docs/SELENIUM-GRID.md]: This is a limitation of EXISTING controller type. No reason to document it here.

File: src/main/resources/seleniumGrid.properties (hunk 1 above)
Comment (adini121) [inline src/main/resources/seleniumGrid.properties]: No not really, I just could not find the entry point where I could provide the grid instance, perhaps you could please link me to it?

**Comment by adini121** [comment]
Understood, will rectify that.

File: docs/SELENIUM-GRID.md (hunk 2 above)
Comment (adini121) [inline docs/SELENIUM-GRID.md]: I did not find this limitation in CONTROLLER.md, hence I added it. I guess I'll remove it from here then.

File: src/main/resources/seleniumGrid.properties (hunk 1 above)
Comment (olivergondza) [inline src/main/resources/seleniumGrid.properties]: https://github.com/jenkinsci/acceptance-test-harness/blob/b27c150f35a386dfced4a5960127bf62f9c34363/src/main/java/org/jenkinsci/test/acceptance/machine/JcloudsMachineProvider.java#L364-375

**Pull request closed without merging**